README.md

```diff
@@ -13,39 +13,41 @@ OpenMC, Matplotlib, NumPy, PySide2
 
 ## Usage
 
-From a directory containing an OpenMC model run:
+From a directory containing a working OpenMC model run:
 
 ```console
-python /path/to/plot_explorer.py
+python /path/to/openmc-plotter
 ```
 
+Press `<kbd>Shift</kbd>+?` for a variety of keyboard shortcuts.
```

Comment:
Should it just say "?" as opposed to "Shift+?"

Reply:
Yeah that's more consistent with what's in the GUI itself.